MorphicServer/Database.cs

```diff
@@ -53,47 +53,41 @@ public class DatabaseSettings
     /// <summary>A connection to the Morphic database</summary>
     public class Database
     {
+        /// <summary>The MongoDB client connection</summary>
+        private readonly MongoClient _client;
```

Comment:
Do we need the leading underscore on these names?  It's not really what we do anywhere else, is it?  I get changing to lowercase because I did a bad job standardizing on lowercase for private fields.

Reply:
Hm. Good point. My ide (rider) suggests this but I bet I can change those templates. Let me see.

Reply:
I found the setting in Rider. Will push a fix for this.